src/libs/workspace-utils.ts

```diff
@@ -159,10 +159,26 @@ export const isValidWsExportTarget = safeCurry((sourceWs: WorkspaceWrapper, dest
   );
 });
 
-// Returns a message explaining why the user can't edit the workspace, or undefined if they can
-export const editWorkspaceError = ({ accessLevel, workspace: { isLocked } }) => {
-  return cond(
-    [!canWrite(accessLevel), () => 'You do not have permission to modify this workspace.'],
-    [isLocked, () => 'This workspace is locked.']
+export const canEditWorkspace = ({
+  accessLevel,
+  workspace: { isLocked },
+}: {
+  accessLevel: WorkspaceAccessLevel;
+  workspace: { isLocked: boolean };
+}) =>
```

Comment:
it's very valuable to specify the return type on function signatures. So lets get those in for these two exports.

Reply:
https://github.com/DataBiosphere/terra-ui/pull/4268/commits/0cd34e362bca1a253c51b4b27701a724f67c9605